readthedocs/extra/basic/working-with-updates.rst

```diff
@@ -121,6 +121,30 @@ random number, while if you say ``'eval 4+4'``, you will reply with the
 solution. Try it!
 
 
+Stopping propagation of Updates
+*******************************
+
+There might be cases when an event handler is supposed to be used solitary and
+it makes no sense to process any other handlers in the chain. For this case,
+it is possible to raise a ``StopPropagation`` exception which will cause the
+propagation of the update through your handlers to stop:
+
+    .. code-block:: python
+
+        @client.on(events.NewMessage)
+        def _(event):
+            # ... some conditions
+            event.delete()
+
+            # Other handlers won't have an event to work with
+            raise client.StopPropagation
```

Comment:
I don't really like accessing a *class* attribute from an *instance* of the `TelegramClient`. It doesn't make any sense.

Reply:
In this case I didn't want to add another import to the docs to keep it simpler, but I guess the import is necessary, depending on what you respond to the comment above.

Reply:
Stopping propagation is not something I would call "basic" usage, so I feel like another import won't make much harm.